data/testimonials.json

```diff
@@ -22,6 +22,49 @@
     "1407215334327603200",
     "1407523834069086208",
     "1407884276381609989",
-    "1407979501754011654"
```

Comment:
Just missing a comma

Reply:
Nah, this was the previous last line and it's been replaced by itself with a comma - see the additions. I always make sure to validate json files before committing them =)

Reply:
@theyokohamalife can you approve? Let me know if there are any actual changes you want.